src/Rules/Functions/MissingFunctionParameterTypehintRule.php

```diff
@@ -0,0 +1,75 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Rules\Functions;
+
+use PhpParser\Node;
+use PHPStan\Analyser\Scope;
+use PHPStan\Broker\Broker;
+use PHPStan\Reflection\BrokerAwareExtension;
+use PHPStan\Reflection\FunctionReflection;
+use PHPStan\Reflection\ParameterReflection;
+use PHPStan\Type\MixedType;
+
+final class MissingFunctionParameterTypehintRule implements \PHPStan\Rules\Rule, BrokerAwareExtension
+{
+
+	/** @var Broker */
+	private $broker;
+
+	/**
+	 * @return string Class implementing \PhpParser\Node
+	 */
+	public function getNodeType(): string
+	{
+		return \PhpParser\Node\Stmt\Function_::class;
+	}
+
+	/**
+	 * @param \PhpParser\Node\Stmt\Function_ $node
+	 * @param \PHPStan\Analyser\Scope $scope
+	 *
+	 * @return string[] errors
+	 */
+	public function processNode(Node $node, Scope $scope): array
+	{
+		if ($scope->isInClass()) {
+			throw new \PHPStan\ShouldNotHappenException();
+		}
+
+		$functionReflection = $this->broker->getCustomFunction(new Node\Name($node->name, $node->getAttributes()), $scope);
+
+		$messages = [];
+
+		foreach ($functionReflection->getParameters() as $parameterReflection) {
+			$message = $this->checkFunctionParameter($functionReflection, $parameterReflection);
+			if ($message === null) {
+				continue;
+			}
+
+			$messages[] = $message;
+		}
+
+		return $messages;
+	}
+
+	private function checkFunctionParameter(FunctionReflection $functionReflection, ParameterReflection $parameterReflection): ?string
+	{
+		$parameterType = $parameterReflection->getType();
+
+		if ($parameterType instanceof MixedType && !$parameterType->isExplicitMixed()) {
+			return sprintf(
+				'Function %s() has parameter $%s with no typehint specified',
```

Comment:
Please add a dot `.` at the end. If there are other rules from you without this trailing dot, please add it in a new commit as well 😊